modules/nw-allocate-load-balancers-to-subnets-procedure.adoc

```diff
@@ -0,0 +1,124 @@
+
+** You have sufficient available IP addresses within these subnets for all assigned roles.
+
+** The AWS configuration for these subnets, including network ACLs and security groups, must permit necessary traffic for all roles assigned to them. For subnets hosting an ingress controller, this typically includes TCP ports 80 and 443 from required sources.
+
+You have the OpenShift installer binary for your target OpenShift version.
+
+* You have an `install-config.yaml` file.
+
+.Procedure
+
+. Prepare the `install-config.yaml` file:
++
+If you haven't already, generate the installation configuration file using the OpenShift installer:
++
+[source,terminal]
+----
+$ openshift-install create install-config --dir=<your_installation_directory>
+----
++
+This command creates the `install-config.yaml` file in the specified directory.
+
+. Define Subnets and assign roles:
++
+Open the `install-config.yaml` file located in `<your_installation_directory>` using a text editor. You will define your VPC subnets and their designated roles under the `platform.aws.vpc.subnets` field.
++
+For each AWS subnet you intend the cluster to use, you will create an entry specifying its `id` and a list of `roles`. Each role is an object with a `type` key. To designate a subnet for the default Ingress Controller, assign it a role with `type: IngressControllerLB`.
++
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com <1>
+metadata:
+  name: my-cluster # Example cluster name
+platform:
+  aws:
+    region: us-east-1 <2>
+    vpc: <3>
+      subnets: <4>
+      - id: subnet-0fcf8e0392f0910d5 # Public Subnet in AZ us-east-1a <5>
+        roles:
+        - type: IngressControllerLB <6>
+        - type: BootstrapNode
+      - id: subnet-0xxxxxxxxxxxxxxza # Public Subnet in another AZ for HA
+        roles:
+        - type: IngressControllerLB
+      - id: subnet-0fcf8e0392f0910d4 # Private Subnet in AZ us-east-1a
+        roles:
+        - type: ClusterNode <7>
+      - id: subnet-0yyyyyyyyyyyyyyzb # Private Subnet in another AZ for HA
+        roles:
+        - type: ClusterNode
+      # Add other subnet IDs and their roles as needed for your cluster architecture
+pullSecret: '...' <8>
+sshKey: '...' <9>
+----
+<1> Your base domain.
+<2> Your AWS region.
+<3> The vpc object under `platform.aws` contains the subnets list.
+<4> List of all subnet objects that OpenShift will use. Each object defines a subnet id and its roles.
+<5> Replace with your AWS Subnet ID. 
+<6> The `type: IngressControllerLB` role specifically designates this subnet for the default Ingress Controller's LoadBalancer. In private/internal cluster, the subnet with `IngressControllerLB` role must be private.
+<7> The `type: ClusterNode` role designates this subnet for control plane and compute nodes. These are typically private subnets.
+<8> Your pull secret.
+<9> Your SSH key.
++
+Entries for Control Plane load balancers in the `subnets` list would follow a similar pattern:
```

Comment:
Could we use consistent word for the `Control Plane` LB or `API` LB ?  User might be confused  with the two words.